specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4275,7 +4446,7 @@
         }
       },
       "required": [
-          "@odata.type", "description", "context", "inputs", "outputs"
```

Comment:
So this is a bit weird... description and context are truly optional...

inputs and outputs are technically optional too, but if left as null for any skill (save the `WebApiSkill`) the API will throw a validation error... I am unsure how to model that in swagger...

Reply:
Will fix; Thanks for confirming